source/MaterialXView/OpenGL/GLUtilityContext.h

```diff
@@ -71,8 +72,6 @@ class GLUtilityContext
   protected:
     /// Create the base context. A OpenGL context to share with can be passed in.
     GLUtilityContext(const WindowWrapper& windowWrapper, HardwareContextHandle context = 0);
-    /// Default destructor
-    virtual ~GLUtilityContext();
 
     /// Base context singleton
     static GLUtilityContext* _globalGLUtilityContext;
```

Comment:
This could be a shared pointer, to make sure the singleton has ownership of it (as comments above).